src/main/java/com/lohika/morning/ecs/vaadin/EcsLabel.java

```diff
@@ -0,0 +1,61 @@
+package com.lohika.morning.ecs.vaadin;
+
+import com.vaadin.data.HasValue;
+import com.vaadin.shared.Registration;
+import com.vaadin.shared.ui.ContentMode;
+import com.vaadin.ui.Label;
+import org.apache.commons.lang3.StringUtils;
+
+import static com.lohika.morning.ecs.utils.EcsUtils.formatString;
+
+public class EcsLabel extends Label implements HasValue<String> {
```

Comment:
By the name of the class it is unclear what he is doing

Reply:
Just a custom (ECS) label implementation